src/System.Net.Security/tests/FunctionalTests/ClientAsyncAuthenticateTest.cs

```diff
@@ -27,20 +27,14 @@ public class ClientAsyncAuthenticateTest
 
         private static readonly SslProtocols[] s_eachSslProtocol = new SslProtocols[]
         {
-            SslProtocols.Ssl2,
             SslProtocols.Ssl3,
             SslProtocols.Tls,
             SslProtocols.Tls11,
             SslProtocols.Tls12,
         };
 
-        public static IEnumerable<object[]> ProtocolMismatchData()
+        private static IEnumerable<object[]> ProtocolMismatchData()
```

Comment:
I'm surprised this works, on Linux I've always found that \[MemberData\] required public members.  Maybe there was a CLR bug that has been fixed since the last time I tried private...